Essentials/src/com/earth2me/essentials/commands/Commandignore.java

```diff
@@ -44,7 +45,15 @@ protected void run(final Server server, final User user, final String commandLab
                 user.sendMessage(tl("unignorePlayer", player.getName()));
             } else {
                 user.setIgnoredPlayer(player, true);
-                user.sendMessage(tl("ignorePlayer", player.getName()));
+                if (args.length > 1) {
+                    long ignoreTimestamp = 0;
+                    final String time = args[1];
+                    ignoreTimestamp = DateUtil.parseDateDiff(time, true);
+                    user.setIgnoredPlayerTimeout(player, ignoreTimestamp);
+                    user.sendMessage(tl("ignorePlayerFor", player.getName(), time));
```

Comment:
Properly format the message to the player and remove timestamp initialization.
```suggestion
                    long ignoreTimestamp = DateUtil.parseDateDiff(time, args[1]);
                    user.setIgnoredPlayerTimeout(player, ignoreTimestamp);
                    user.sendMessage(tl("ignorePlayerFor", player.getName(), DateUtil.formatDateDiff(ignoreTimestamp)));
```